addons/mail/static/src/core/common/quick_reaction_menu.js

```diff
@@ -51,7 +54,7 @@ export class QuickReactionMenu extends Component {
 
     openPicker() {
         this.dropdown.close();
-        this.picker.open(this.toggle.el, { onSelect: this.toggleReaction.bind(this) });
+        this.picker.open(this.toggle);
```

Comment:
strange API to take a `ref` as param rather than element, especially when all internal usages are `ref.el`